src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -577,6 +577,11 @@ public static void setup(SlimefunPlugin plugin) {
 		new ItemStack[] {new CustomItem(SlimefunItems.STEEL_INGOT, 8), null, null, null, null, null, null, null, null})
 		.register(plugin);
 
+		new SlimefunItem(categories.resources, SlimefunItems.LEAD_PLATE, RecipeType.COMPRESSOR, new ItemStack[] {
```

Comment:
To be consistent with the other plates: This recipe should also be added to the Electric Press.